src/test/java/fi/aalto/cs/apluscourses/presentation/ViewModelExtensions.java

```diff
@@ -12,11 +13,17 @@ private ViewModelExtensions() {
 
   public static class TestNodeViewModel extends SelectableNodeViewModel<Object> {
     private final long id;
+    private final boolean hideByDefault;
```

Comment:
Since the test was removed perhaps the variable `hideByDefault` could also be removed?

Reply:
oops I forgot